validator/main.go

```diff
@@ -693,6 +693,13 @@ func isDriverManagedByOperator(ctx context.Context) (bool, error) {
 
 func validateHostDriver(silent bool) error {
 	log.Info("Attempting to validate a pre-installed driver on the host")
+	fileInfo, err := os.Lstat("/host/usr/bin/nvidia-smi")
+	if err != nil {
+		return fmt.Errorf("no 'nvidia-smi' file present on the host")
```

Comment:
Let's log the error too.

Reply:
A separate log statement or append the error message to the error that gets returns here?

Reply:
the latter. Let's wrap the error

Reply:
Done.